spec/Overview.html

```diff
@@ -831,8 +831,9 @@ <h4>The getRandomValues method</h4>
             <ol>
               <li>
                 <p>
-                  If <var>array</var> is not of an integer type (i.e., Int8Array, Uint8Array,
-                  Int16Array, Uint16Array, Int32Array, Uint32Array or Uint8ClampedArray), [= exception/throw =] a
+                  If <var>array</var> is not an Int8Array, Uint8Array, Uint8ClampedArray,
+                  Int16Array, Uint16Array, Int32Array, Uint32Array,
+                  BigInt64Array or BigUint64Array, [= exception/throw =] a
```

Comment:
Would we be able to use `{{ }}` for these array references? I believe the Web IDL spec is exporting these terms.

Reply:
Hmhm, good idea, thanks :+1: